src/sphinx_tags/__init__.py

```diff
@@ -86,17 +86,22 @@ def create_file(self, items, extension, tags_output_dir, srcdir):
             list of file extensions used.
         srcdir : str
             root folder for the documentation (usually, project/docs)
+        tags_page_title: str
+            the title of the tag page, after which the tag is listed (e.g. "Tag: ...")
+        tags_page_header: str
+            the words after which the pages with the tag are listed, e.g. "With this tag")
+
 
         """
         content = []
         if "md" in extension:
             filename = f"{self.name}.md"
-            content.append(f"# Tag: {self.name}")
+            content.append(f"# {tags_page_title}: {self.name}")
             content.append("")
             content.append("```{toctree}")
-            content.append("---")
+            content.append("--------")
```

Comment:
This change is actually unnecessary; the syntax for MyST is actually for this separator to match the one on line 105 below, so it should be just 3 dashes as originally.
```suggestion
            content.append("---")
```

Reply:
Gotcha! :)